include/z64player.h

```diff
@@ -180,6 +180,123 @@ typedef enum {
+    /*  3 */ PLAYER_MODELGROUP_DEFAULT, // non-specific models, for items that don't have particular link models
+    /*  4 */ PLAYER_MODELGROUP_4, // unused, same as PLAYER_MODELGROUP_DEFAULT
+    /*  5 */ PLAYER_MODELGROUP_BGS, // biggoron sword
+    /*  6 */ PLAYER_MODELGROUP_BOW_SLINGSHOT, // bow/slingshot
+    /*  7 */ PLAYER_MODELGROUP_EXPLOSIVES, // bombs, bombchus, same as PLAYER_MODELGROUP_DEFAULT
+    /*  8 */ PLAYER_MODELGROUP_BOOMERANG,
+    /*  9 */ PLAYER_MODELGROUP_HOOKSHOT,
+    /* 10 */ PLAYER_MODELGROUP_10, // stick/fishing pole (which are drawn separately)
+    /* 11 */ PLAYER_MODELGROUP_HAMMER,
+    /* 12 */ PLAYER_MODELGROUP_OCARINA, // ocarina
+    /* 13 */ PLAYER_MODELGROUP_OOT, // ocarina of time
+    /* 14 */ PLAYER_MODELGROUP_BOTTLE, // bottles (drawn separately)
+    /* 15 */ PLAYER_MODELGROUP_15, // "last used"
+    /* 16 */ PLAYER_MODELGROUP_MAX
+} PlayerModelGroup;
+
+typedef enum {
+    /* 0 */ PLAYER_MODELGROUPENTRY_ANIM,
+    /* 1 */ PLAYER_MODELGROUPENTRY_LEFT_HAND,
+    /* 2 */ PLAYER_MODELGROUPENTRY_RIGHT_HAND,
+    /* 3 */ PLAYER_MODELGROUPENTRY_SHEATH,
+    /* 4 */ PLAYER_MODELGROUPENTRY_WAIST,
+    /* 5 */ PLAYER_MODELGROUPENTRY_MAX
+} PlayerModelGroupEntry;
+
+typedef enum {
+    // left hand
+    /*  0 */ PLAYER_MODELTYPE_LH_OPEN, // empty open hand
+    /*  1 */ PLAYER_MODELTYPE_LH_CLOSED, // empty closed hand
+    /*  2 */ PLAYER_MODELTYPE_LH_SWORD, // holding kokiri/master sword
+    /*  3 */ PLAYER_MODELTYPE_3, // unused, same as PLAYER_MODELTYPE_LH_SWORD
+    /*  4 */ PLAYER_MODELTYPE_LH_BGS, // holding bgs/broken giant knife (child: master sword)
+    /*  5 */ PLAYER_MODELTYPE_LH_HAMMER, // holding hammer (child: empty hand)
+    /*  6 */ PLAYER_MODELTYPE_LH_BOOMERANG, // holding boomerang (adult: empty hand)
+    /*  7 */ PLAYER_MODELTYPE_LH_BOTTLE, // holding bottle (bottle drawn separately)
+    // right hand
+    /*  8 */ PLAYER_MODELTYPE_RH_OPEN, // empty open hand
+    /*  9 */ PLAYER_MODELTYPE_RH_CLOSED, // empty closed hand
+    /* 10 */ PLAYER_MODELTYPE_RH_SHIELD, // holding a shield (including no shield)
+    /* 11 */ PLAYER_MODELTYPE_RH_BOW_SLINGSHOT, // holding bow/slingshot
+    /* 12 */ PLAYER_MODELTYPE_12, // unused, same as PLAYER_MODELTYPE_RH_BOW_SLINGSHOT
+    /* 13 */ PLAYER_MODELTYPE_RH_OCARINA, // holding ocarina (child: fairy ocarina, adult: OoT)
+    /* 14 */ PLAYER_MODELTYPE_RH_OOT, // holding OoT
+    /* 15 */ PLAYER_MODELTYPE_RH_HOOKSHOT, // holding hookshot (child: empty hand)
+    // sheath
+    /* 16 */ PLAYER_MODELTYPE_SHEATH_16, // sheathed kokiri/master sword?
+    /* 17 */ PLAYER_MODELTYPE_SHEATH_17, // empty sheath?
+    /* 18 */ PLAYER_MODELTYPE_SHEATH_18, // sword sheathed and shield on back?
+    /* 19 */ PLAYER_MODELTYPE_SHEATH_19, // empty sheath and shield on back?
+    // waist
+    /* 20 */ PLAYER_MODELTYPE_WAIST,
+    /* 21 */ PLAYER_MODELTYPE_MAX,
+    /* 0xFF */ PLAYER_MODELTYPE_RH_FF = 0xFF // disable shield collider, cutscene-specific
```

Comment:
I'd personally just go with hex for all these enums

Reply:
I'll use hex from now on or change it without debating if commented on